app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,15 @@ class PostsController < ApplicationController
   # GET /posts.json
   def index
     @posts = Post.all
+    @posts_done_homework = Post.where(done_homework: true, user_id: current_user) 
+    @posts_pending_homework =  Post.where(done_homework: false, user_id: current_user) 
```

Comment:
Same here...
Oh, what a coincidence... I saw the _done_homework_ variable in another project 😅